content/docs/connect/connection-pooling.md

```diff
@@ -7,6 +7,10 @@ redirectFrom:
 
 Each PostgreSQL connection creates a new process in the operating system, which consumes resources. For this reason, PostgreSQL limits the number of open connections. Neon permits 100 simultaneous PostgreSQL connections, by default, with a `max_connections=100` setting, which is the typical default for this parameter. A small number of those connections are reserved for administrative purposes. A connection limit of 100 may not be sufficient for some applications. To increase the number of connections that Neon supports, you can enable connection pooling.
 
+<Admonition type="note">
+Connection pooling is not yet supported with our [Autoscaling](../introduction/autoscaling) feature, which is currently in Beta. If you expect a large number of concurrent connections, we recommend using a **Fixed size** compute (the default), which supports connection pooling. For compute configuration instructions, see [Compute size and Autoscaling configuration](../manage/endpoints#compute-size-and-autoscaling-configuration).
```

Comment:
This is minor, but perhaps we can shorten the
'Autoscaling feature, which is currently in Beta' to 'Autoscaling (Beta)' or something along these lines?
would be great to be able to annotate with the Beta labels directly in the docs.